packages/react-native-windows-init/src/Cli.ts

```diff
@@ -222,6 +230,35 @@ function isProjectUsingYarn(cwd: string) {
 
     const rnwResolvedVersion = await getLatestMatchingRNWVersion(version);
 
+    if (!rnwResolvedVersion) {
+      if (!argv.version) {
```

Comment:
nit: kinda subjective, but I think this is a bit more readable if you reverse the ordering of the blocks and do `if (argv.version)`